versions/3.0.3.md

```diff
@@ -1891,13 +1891,13 @@ $request.body#/successUrls/2 | http://clientdomain.com/medium
 $response.header.Location | http://example.org/subscription/1
 
 
-##### Callback Object Example
+##### Callback Object Examples
 
-The following example shows a callback to the URL specified by the `id` and `email` property in the request body.
+The following example uses the user provided `queryUrl` query string parameter to define the callback URL.
```

Comment:
```suggestion
The following example uses the user provided `queryUrl` query string parameter to define the callback URL.  This is an example of how to use callback objects to describe a WebHook callback that goes with the subscription operation to enable registering for the callback.
```
@philsturgeon How do you feel about adding this?  I'm ok with taking the webhook word out of the actual examples.

Reply:
Ship it!